apps/organisations/views.py

```diff
@@ -215,44 +218,76 @@ def get_context_data(self, **kwargs):
 
     def form_valid(self, form):
         data = form.cleaned_data
-        img = self.generate_image(data)
         context = self.get_context_data()
+        img = self.generate_image(data)
+        # get the content form with unchanged cleaned data, as image generation
+        # changes the image data (why?)
+        content_form = CommunicationContentCreationForm(
+            initial=data, project=self.project, format=self.format)
```

Comment:
@Rineee I still get weird error?! I will add all the correct configs and then look at this more

Reply:
@phillimorland Hm, I checked again and for me it still works.. did you check with only these commits?